File: net/management-api-v2/cm_api_v2_patch_environment.cs
// Tip: Find more about .NET SDKs at https://kontent.ai/learn/net
using Kontent.Ai.Management;
using Kontent.Ai.Management.Models.Environments.Patch;

var client = new ManagementClient(new ManagementOptions
{
    ApiKey = "KONTENT_AI_MANAGEMENT_API_KEY",
    EnvironmentId = "KONTENT_AI_ENVIRONMENT_ID"
});

var response = await client.ModifyEnvironmentAsync(new[]
{
        new EnvironmentRenamePatchModel
        {
            Value = "My Little Production"
        }
});

File: ts/management-api-v2/cm_api_v2_patch_environment.ts
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
import { ManagementClient } from '@kontent-ai/management-sdk';

const client = new ManagementClient({
  environmentId: 'KONTENT_AI_ENVIRONMENT_ID',
  apiKey: 'KONTENT_AI_MANAGEMENT_API_KEY'
});

const response = await client.modifyEnvironment()
  .withData(
    [
      {
        op: 'rename_environment',
        value: 'My Little Production'
      }
    ]
  )
  .toPromise();